src/pages/underConstruction/construction.ts

```diff
@@ -15,6 +17,11 @@ export class ConstructionPage {
         this.navCtrl = navCtrl;
         this.pageTitle = navParams.get('linkParam');
     }
+
+    openLink(){
```

Comment:
can we just define the function once globally somewhere (take in the doc url as a parameter) rather than the same function defined in multiple places?